pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -130,8 +62,6 @@
   - elsif rack_env != :production && request.params["lock-hero"]
     -# for UI tests just lock to the first hero image
     - heroes_arranged = heroes[0, 1]
-  - elsif show_hoc_launch_layout
-    - heroes_arranged = hoc_launch_hero[hoc_launch_hero_variation]
   - elsif show_census_layout
```

Comment:
The idea is that we don't know what the homepage takeover will be for next year so we should gut anything related to hoc? Just confirming there isn't anything reusable.

Reply:
That was my thinking.  I think we could generalise this technique going forward, but it should probably be done in conjunction with a rethink about the gsheet-driven approach for driving the homepage in general.